src/Controller/AppController.php

```diff
@@ -382,6 +382,8 @@ protected function applySessionFilter(): ?Response
             $session->write($sessionKey, $this->request->getQueryParams());
 
             return null;
+        } else { // empty? remove session key
+            $session->delete($sessionKey);
```

Comment:
This way session filter is not working... every URL with empty query string will cause session key removal